ballerina/tests/test_client.bal

```diff
@@ -0,0 +1,693 @@
+    test:assertEquals(res, ());
+}
+
+@test:Config {}
+function testPostCalendarList() returns Error?  {
+    Client client1 = check new(config);
+    string summary = "Test Calendar List Entry";
+    Calendar cal = {
+        summary: summary
+    };
+    Calendar calendar = check client1->/calendars.post(cal);
+    test:assertEquals(calendar.summary, summary);
+    CalendarListEntry calendarListEntry = {
+        id: calendar.id
+    };
+    CalendarListEntry calendarUpdate = check client1->/users/me/calendarList.post(calendarListEntry);
+    test:assertEquals(calendarUpdate.summary, summary);
+
+    Error? res = check client1->/users/me/calendarList/[<string>calendarUpdate.id].delete();
+    test:assertEquals(res, ());
+}
+
+@test:Config {}
+function testGetCalendarListEntry() returns Error?  {
+    Client client1 = check new(config);
+    string summary = "Test Calendar List Entry";
+    Calendar cal = {
+        summary: summary
+    };
+    Calendar calendar = check client1->/calendars.post(cal);
+    test:assertEquals(calendar.summary, summary);
+    CalendarListEntry calendarListEntry = {
+        id: calendar.id
+    };
+    CalendarListEntry createdCalendarListEntry = check client1->/users/me/calendarList.post(calendarListEntry);
+    CalendarListEntry retrievedCalendarListEntry = check client1->/users/me/calendarList/[<string>createdCalendarListEntry.id].get();
+    test:assertEquals(retrievedCalendarListEntry.summary, summary);
+    Error? res = check client1->/users/me/calendarList/[<string>createdCalendarListEntry.id].delete();
+    test:assertEquals(res, ());
+}
+
+@test:Config{}
+function testUpdateCalendarListEntry() returns Error?  {
+    Client client1 = check new(config);
+    string summary = "Test Calendar List Entry";
+    Calendar cal = {
+        summary: summary
+    };
+    Calendar calendar = check client1->/calendars.post(cal);
+    test:assertEquals(calendar.summary, summary);
+
+    CalendarListEntry calendarListEntry = {
+        id: calendar.id
+    };
+    CalendarListEntry calendarUpdate = check client1->/users/me/calendarList.post(calendarListEntry);
+    test:assertEquals(calendarUpdate.summary, summary);
+
+    CalendarListEntry updatedEntry = check client1->/users/me/calendarList/[<string>calendarUpdate.id].put(calendarListEntry);
+    test:assertEquals(updatedEntry.id, calendarUpdate.id);
+
+    Error? res = check client1->/users/me/calendarList/[<string>calendarUpdate.id].delete();
+    test:assertEquals(res, ());
+}
+
+@test:Config{}
+function testGetCalendarList() returns Error?  {
+    Client client1 = check new(config);
+    CalendarList calendarList = check client1->/users/me/calendarList.get();
+    test:assertNotEquals(calendarList, ());
+    CalendarListEntry[]? calendarListEntries = calendarList.items;
+    if calendarListEntries !is () {
+        test:assertTrue(calendarListEntries.length() > 0);
+    }

```

Comment:
```suggestion
    test:assertTrue(calendarListEntries is () || calendarListEntries.length() > 0);

```

Reply:
Fixed in [5ec2e48](https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/pull/134/commits/5ec2e48c2cbc1fa15e119f5cee8d8021de16eae5)